h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -99,15 +106,44 @@ function TextField({
 }
 
 export default function CreateGroupForm() {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const config = readConfig();
```

Comment:
```suggestion
  const config = useMemo(() => readConfig(), [])
```

This avoids re-reading the config on each render.

Reply:
Done